### ISSUE TITLE
[v16] Ciritcal CSS for SSR

### ISSUE THREAD

**Issue by chymz** [vuejs/vue-loader#1759]
### What problem does this feature solve?
Enable cirtical CSS in SSR mode: https://github.com/vuejs/vue-loader/blob/next/src/index.ts#L203

Do you have any updates on this ?

### What does the proposed API look like?
Like for Vue 2 ?

**Comment by camwhite** [comment]
Any movement here?

**Comment by LinusBorg** [comment]
No.

**Comment by glopezep** [comment]
.................

**Comment by pdanpdan** [comment]
The Todo look like it is after the if for modules/not modules, so it should be that issue. Anyway, it's not from quasar.

**Comment by chetzof** [comment]
Does this issue relate to the fact that CSS modules are not generated on server, only after the component is rerendered on client? Or should I open a new issue?

**Comment by lifeishell** [comment]
I add a pr for this https://github.com/vuejs/vue-loader/pull/1855

**Comment by gfordinal** [comment]
Any movement here?
Still having the issue <style module> not rendering in ssr mode.
